tidy3d/web/core/log.py

```diff
@@ -0,0 +1,364 @@
+"""Logging for core."""
```

Comment:
is this copied from our regular `tidy3d/log.py` file? I think it might be easier to simply use the regular `logging` module for web/core errors and logging 

https://docs.python.org/3/library/logging.html

Reply:
modified